src/libraries/System.Security.Cryptography.Cose/ref/System.Security.Cryptography.Cose.cs

```diff
@@ -137,6 +136,7 @@ internal CoseSign1Message() { }
         public bool VerifyDetached(System.Security.Cryptography.AsymmetricAlgorithm key, byte[] detachedContent, byte[]? associatedData = null) { throw null; }
         public bool VerifyDetached(System.Security.Cryptography.AsymmetricAlgorithm key, System.IO.Stream detachedContent, System.ReadOnlySpan<byte> associatedData = default(System.ReadOnlySpan<byte>)) { throw null; }
         public bool VerifyDetached(System.Security.Cryptography.AsymmetricAlgorithm key, System.ReadOnlySpan<byte> detachedContent, System.ReadOnlySpan<byte> associatedData = default(System.ReadOnlySpan<byte>)) { throw null; }
+        public bool VerifyDetached(System.Security.Cryptography.Cose.CoseKey key, byte[] detachedContent, byte[]? associatedData = null) { throw null; }
```

Comment:
Are these new overloads being given the same treatment as the AsymmetricAlgorithm-based ones?

I don't know if the AsymmetricAlgorithm ones just test the array ones and let that fall back to the span ones by ccov/inspection, or if they are testing them independently.  Since I don't see these new methods being wired into tests (except the ArgumentNullException ones) it means this set is either testing span-only, or array-only (which, yes, hits the span ones by inspection/ccov).